wpilibc/src/main/native/cpp/ADXRS450_Gyro.cpp

```diff
@@ -14,7 +14,7 @@
 
 using namespace frc;
 
-static constexpr double kSamplePeriod = 0.0005;
+static constexpr units::second_t kSamplePeriod{0.0005};
 static constexpr double kCalibrationSampleTime = 5.0;
```

Comment:
Do you want to do this one too?

Reply:
We can't unless I do a units overload for Wait().